components/Tabs/tests/unit/MDCItemBarTests.m

```diff
@@ -110,6 +110,30 @@ - (void)testItemBarViewIncludesTabInAccessibilityLabelOniOS9 {
   }
 }
 
+- (void)testItemBarItemSetAccessibilityLabel {
```

Comment:
This test looks good for the initial value. Would you please add one more test that changes the `UITabBarItem.accessibilityLabel` after the items are assigned to the item bar? I believe that would be a good test of the KVO logic.